packages/compiler-dom/src/transforms/stringifyStatic.ts

```diff
@@ -356,7 +356,7 @@ function stringifyElement(
 // (see compiler-core/src/transforms/transformExpression)
 function evaluateConstant(exp: ExpressionNode): string {
   if (exp.type === NodeTypes.SIMPLE_EXPRESSION) {
-    return new Function(`return ${exp.content}`)()
+    return new Function(`return ${removeEscapes(exp.content)}`)()
```

Comment:
```suggestion
    return new Function(`return (${exp.content})`)()
```

I don't think it's necessary to use a regexp here.